LaTeX/TeX.sublime-syntax

```diff
@@ -277,30 +272,51 @@ contexts:
     - include: paragraph-pop
 
   def-definition:
-    - match: \{
-      scope: punctuation.definition.group.brace.begin.tex
-      set: def-definition-body
+    - match: (?=\{)
+      set:
+        - def-scope-as-body
+        - macro-braces-begin
```

Comment:
This pattern can directly replace line 265-267 to avoid duplicate lookahead evaluation.

The whole `def-definition` context becomes somewhat obsolete